pyproject.toml

```diff
@@ -66,6 +66,39 @@ all = [
     "pydantic-settings >=2.0.0",
     "pydantic-extra-types >=2.0.0",
 ]
+httpx = [
+    "httpx >=0.23.0",
+]
+jinja = [
+    "jinja2 >=2.11.2",
+]
+python-multipart = [
+    "python-multipart >=0.0.5",
```

Comment:
Python-multipart has been bumped to `>=0.0.7` so this needs to be updated here too.

Reply:
Updated, thanks for reminding!